x-pack/plugins/maps/common/descriptor_types/source_descriptor_types.ts

```diff
@@ -167,3 +167,23 @@ export type GeojsonFileSourceDescriptor = {
   name: string;
   type: string;
 };
+
+export type TableSourceColumn = {
```

Comment:
This is the same type as `GeoJsonFileFieldDescriptor` how about combining into a single type? They are conveying the same information, just one a property on a geojson object and another is a propery in a row